### PR TITLE
fix: set `displayname` and `__ui__`

### LINKED ISSUE #2895: `displayName` is not set for the component of "@yamada-ui/rating"
script: component-display-name
package: @yamada-ui/rating

### Description
The component "@yamada-ui/rating" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

### PR DESCRIPTION
**Pull request by darkcat666**
Closes #2895 

## Description

The component "@yamada-ui/rating" uses forwardRef, but displayName is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

## New behavior

Set `displayNane` and `__ui__`

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/rating/src/rating-item.tsx
Comment (illionillion) [inline packages/components/rating/src/rating-item.tsx:1]: @darkcat666 

Thank for PR!!

Please import `FC` from `@yamada-ui/core`.
StarIcon and RatingIcon should be modified in the same way.

**Review by illionillion — CHANGES_REQUESTED** [review]

File: packages/components/rating/src/rating-item.tsx
Comment (darkcat666) [reply packages/components/rating/src/rating-item.tsx:1]: @illionillion 

I added missing import `FC` from @yamada-ui/core.
Thank you for pointing that out.

**Review by hinataOhyama — APPROVED** [review]

File: .changeset/fuzzy-penguins-hammer.md
Comment (illionillion) [inline .changeset/fuzzy-penguins-hammer.md:1]: Only one changeset is required per package change.

File: packages/components/rating/src/rating-item.tsx
Comment (illionillion) [reply packages/components/rating/src/rating-item.tsx:1]: This has not been corrected.

> StarIcon and RatingIcon should be modified in the same way.

File: .changeset/fuzzy-penguins-hammer.md
Comment (darkcat666) [reply .changeset/fuzzy-penguins-hammer.md:1]: I deleted duplicated changeset.

File: packages/components/rating/src/rating-item.tsx
Comment (darkcat666) [reply packages/components/rating/src/rating-item.tsx:1]: I fixed StarIcon.
Thank you.

Comment (darkcat666) [reply packages/components/rating/src/rating-item.tsx:1]: Sorry, I overlooked What needs to be fixed.
I fixed it for sure this time.

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**